target/linux/lantiq/image/Makefile

```diff
@@ -757,89 +822,104 @@ endif
 ifeq ($(SUBTARGET),falcon)
 
 define Device/lantiq_easy98000-nor
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := NOR
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NOR
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NOR
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nor
 
 define Device/lantiq_easy98000-nand
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := NAND
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NAND
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NAND
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nand
 
 define Device/lantiq_easy98000-sflash
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board SFLASH
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98000SFLASH
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board SFLASH
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-sflash
 
 define Device/lantiq_falcon-mdu
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := MDU Lantiq Falcon / VINAXdp MDU Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := FALCON-MDU
-  DEVICE_TITLE := MDU - Lantiq Falcon / VINAXdp MDU Board
 endef
 TARGET_DEVICES += lantiq_falcon-mdu
 
 define Device/lantiq_easy88388
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY88388 Falcon FTTDP8 Reference Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY88388
-  DEVICE_TITLE := EASY88388 - Lantiq Falcon FTTDP8 Reference Board
 endef
 TARGET_DEVICES += lantiq_easy88388
 
 define Device/lantiq_easy88444
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY88444 Falcon FTTdp G.FAST Reference Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY88444
-  DEVICE_TITLE := EASY88444 - Lantiq Falcon FTTdp G.FAST Reference Board
 endef
 TARGET_DEVICES += lantiq_easy88444
 
 define Device/lantiq_falcon-sfp
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := SFR Falcon SFP Stick
   IMAGE_SIZE := 7424k
   DEVICE_DTS := FALCON-SFP
-  DEVICE_TITLE := SFP - Lantiq Falcon SFP Stick
 endef
 TARGET_DEVICES += lantiq_falcon-sfp
 
 define Device/lantiq_easy98035synce
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98035SYNCE Falcon SFP Stick with Synchronous Ethernet
```

Comment:
This might be a `Falcon SFP Stick with Synchronous Ethernet (EASY98035SYNCE)`?

Reply:
Resolved by `DEVICE_VARIANT := with Synchronous Ethernet` as @dangowrt suggested.